app/models/deletion.rb

```diff
@@ -67,7 +67,7 @@ def purge_fastly
   end
 
   def update_search_index
-    @version.rubygem.delay.update_document
```

Comment:
Is there any alternative to [update_document](https://github.com/elastic/elasticsearch-rails/blob/d12d812c3f52ac484cf73805ef41986dd95ba5a0/elasticsearch-model/lib/elasticsearch/model/indexing.rb#L401-L444) in searchkick @ankane?